packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -55,7 +55,8 @@ impl<C: AsClient> PostgresStore<C> {
 
             let entity = match dependency_status {
                 DependencyStatus::Unresolved => {
-                    <Self as Read<Entity>>::read_into_subgraph(self, subgraph, &entity_edition_id)
+                    subgraph
+                        .get_or_read::<Entity>(self, &entity_edition_id)
```

Comment:
Are there actually situations where this branch _should_ be hit? I'm wondering if we just don't want to allow the `or_read`

Reply:
For ontology types, we may not have inserted an element before traversing it. I expect this to be changed at some point in the near future. For entities you are right, and I already adjusted this in a follow-up PR: https://github.com/hashintel/hash/commit/275f8a87c90d3494afecf868eb33df2588e4a1e3

Reply:
Great, cheers